kernel/packages/shared/apis/EnvironmentAPI.ts

```diff
@@ -53,4 +60,18 @@ export class EnvironmentAPI extends ExposableAPI {
       displayName: `${serverName}-${layer}`
     }
   }
+
+  /**
+   * Returns explorer configuration and environment information
+   */
+  @exposeMethod
+  async getExplorerData(): Promise<ExplorerData> {
```

Comment:
"Data" is a very broad name, just like "Id" what do you think about `getExplorerConfiguration` or something even more specific?

Reply:
Changed.